CONTRIBUTING.md

```diff
@@ -13,24 +13,24 @@ Please note that this project is released with a [Contributor Code of Conduct][c
 
 ## Start with an issue
 
-0. Open an [issue][issue] to discuss the change you want to see.
+1. Open an [issue][issue] to discuss the change you want to see.
 This helps us coordinate and reduce duplication.
-0. Once we've had some discussion, you're ready to code!
+2. Once we've had some discussion, you're ready to code!
```

Comment:
Are we able to add an exception instead of making this change? We tend to use the `1`/`0` formatting on purpose, as it makes it easier to add new items in lists (i.e. we don't have to go change all the other numbers when we put something new in the middle).

Reply:
I've always felt this reasoning was overrated; if someone is making changes to a document anyway, then it's not hard to change a handful of list numbers as well (for relatively short lists). It's can also get a little confusing when working with sub-lists and sub-sub lists, when they are all the same number.

The benefit of proper numbering is that the markdown provides the same information in raw form as in rendered form.

Nevertheless, I have updated the docs to use the form of repeated `1.`, as this form is accepted by the linter (repeated `0.` also works but means that lists start at 0 rather than 1), in commit 1b24d82

Reply:
Thank you!